sdk/chaos/arm-chaos/CHANGELOG.md

```diff
@@ -1,15 +1,12 @@
 # Release History
-
-## 1.0.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
-
+    
+## 1.1.0 (2024-03-06)
+    
+**Features**
+
+  - Interface ExperimentUpdate has a new optional parameter tags
```

Comment:
@RenzoPrettoMS Could you help review this pr? Is the change expected from your side?